firefox_decrypt.py

```diff
@@ -816,7 +833,9 @@ def main():
     # Check if profile is password protected and prompt for a password
     nss.authenticate(args.interactive)
     # Decode all passwords
-    to_export = nss.decrypt_passwords(args.export_pass, args.tabular)
+    to_export = nss.decrypt_passwords(
+        args.export_pass, args.format, csv_delimiter=args.delimiter, csv_quotechar=args.quotechar,
```

Comment:
What exactly do you mean? I think it's the same (dirty) way as it was used before for `tabular`.